core/templates/domain/topic/subtopic.model.ts

```diff
@@ -40,15 +42,14 @@ export class Subtopic {
   _title: string;
   _skillSummaries: ShortSkillSummary[];
   _skillIds: string[];
-  _skillSummary: ShortSkillSummary;
-  _thumbnailFilename: string;
-  _thumbnailBgColor: string;
-  _urlFragment: string;
+  _thumbnailFilename: string | null;
```

Comment:
Ditto -- if null, then there needs to be a clear explanation of what None means and why this can legitimately arise as a "final state" in practice.

Reply:
Same properties as above, I think comment at just one place would be sufficient?